readme.md

```diff
@@ -125,95 +74,92 @@ And here's an example of an unsuccessful `403 Forbidden` API call. Why did it fa
 
 ![Postman screenshot fail](http://i.imgur.com/r3nIhGH.png)
 
-> We'll use Postman more when we test out our own API later in today's class.
-
-
 ## Rails and JSON
 
-### Intro (10 minutes / 0:40)
+### Intro (10 minutes / 0:25)
 
 Today, we're going to use Rails to create our own API from which we can pull information. We will be using a familiar codebase, and modify it so that it can serve up data.  
 
-Let's demonstrate using Tunr.
+Let's demonstrate using Grumblr. Clone down this [starter code](https://github.com/ga-wdi-exercises/grumblr_rails_api/) and checkout the `wdi12-starter` branch...
```

Comment:
people are gonna ask if they can use the existing api they cloned - can they? can you rename wdi12-starter to something more semantic, like api w/o some feature